### ISSUE TITLE
Doesn't work with multi-process

### ISSUE THREAD

**Issue by Keith94** [alphapapa/fabtabs#5]
Tab coloring is broken and the browser console has these errors when opening a tab.
```
unsafe/forbidden CPOW usage RemoteAddonsParent.jsm:872:2
unsafe/forbidden CPOW usage tabtint.js:243
```

**Comment by alphapapa** [comment]
Thanks for reporting this.  If it is simple to fix, I might be interested, but if not, this may never happen--at least, not by me.  Especially with Mozilla throwing out XUL extensions, I don't have a lot of interest in digging into the details.  If it requires a complete rewrite using Jetpack or WebExtensions, then it would probably need to be a new project entirely.  Patches welcome, of course.  :)

**Comment by Keith94** [comment]
You're right, it would eventually require migration to WebExtensions API; many extensions are beginning to do that already.

**Comment by alphapapa** [comment]
Assuming I continue using Firefox after they drop XUL (which I am not sure about), I might look into migrating to it...someday...  :)

**Comment by alphapapa** [comment]
Since I don't plan to rewrite the extension, I'm going to mark this as wontfix, but I'll leave it up as a known issue.  Thanks for reporting it.

**Comment by alphapapa** [comment]
Might as well mark it help-wanted too, just in case anyone wants to take a crack at it...

**Comment by alphapapa** [comment]
Might as well leave it open as long as we still want it.
